src/words/TokTok.jl

```diff
@@ -0,0 +1,172 @@
+# Tok-Tok Tokenizer
+# Replace non-breaking spaces with normal spaces.
+
+const NON_BREAKING = ["\u00A0"], " "
```

Comment:
Use Pairs, written `a=>b`.
They are like tuples but more readable for this kind of thibng

Also for constant collections use tuples over arrays as they are immutable and faster.

```suggestion
const NON_BREAKING = ("\u00A0",) => " "
```